assets/js/src/form_editor/components/form_styling_background.jsx

```diff
@@ -58,6 +58,9 @@ const FormStylingBackground = ({ children }) => {
         borderStyle,
         textAlign,
         padding,
+        width: 700,
+        margin: '0 auto',
+        paddingBottom: 30,
```

Comment:
I think we should drop the paddingBottom here. It overrides padding set by a user in styles settings and when user changes it it gets overridden again. See https://d.pr/v/6Pz3i8. I would respect the padding set in form styles settings.

Reply:
Well, the padding is in the specification. It shouldn't be overridden by user's settings though.

Reply:
Maybe we could ask PM to reconsider the bottom padding, since it is "fighting" with the one user specifies.